src/Components/SearchForm/SearchForm.js

```diff
@@ -6,21 +6,28 @@ class Search extends Component {
 	state = {
 		searchTerm: '',
 	};
-
+	input = React.createRef();
 	handleChange = (e) => {
 		this.setState({ searchTerm: e.target.value });
 	};
 	handleSubmit = (e) => {
 		e.preventDefault();
-		this.state.searchTerm &&
-			this.props.history.push(`/search/photos/${this.state.searchTerm}`);
+		this.state.searchTerm && this.props.className
+			? this.props.history.push(`/search/collections/${this.state.searchTerm}`)
+			: this.props.history.push(`/search/photos/${this.state.searchTerm}`);
```

Comment:
if(this.state.searchTerm && this.props.className){
this.props.history.push(`/search/collections/${this.state.searchTerm}`)
}else {
this.props.history.push(`/search/photos/${this.state.searchTerm}`);
}